.murdock

```diff
@@ -137,7 +144,12 @@ compile() {
     local board=$(echo $2 | cut -f 1 -d':')
     local toolchain=$(echo $2 | cut -f 2 -d':')
 
-    [ "$board" = "makefile_broken" ] && error "$0: Makefile in \"$appdir\" seems to be broken!"
+    [ "$board" = "makefile_broken" ] && {
+        echo "$0: There seems to be a problem in \"$appdir\" while getting supported boards!"
+        echo "$0: testing \"make -C$appdir make info-boards-supported\"..."
```

Comment:
Shouldn't it be `make -C$appdir info-boards-supported`? (Duplicated `make` here)

Reply:
Btw: Feel free to squash right away.

Reply:
fixed & squashed. thx!